lib/ansible/modules/network/aos/aos_login.py

```diff
@@ -114,14 +115,17 @@ def main():
     module = AnsibleModule(
         argument_spec=dict(
             server=dict(required=True),
-            port=dict(default='8888'),
+            port=dict(default='443', type="int"),
```

Comment:
Understood, I agree we need to be careful with default value but the default port changed recently in our product as well.
We think it's better to update it now.